pkg/model/cluster.go

```diff
@@ -149,5 +151,11 @@ func NewCluster(env simulator.Environment, config ClusterConfig, replicasConfig
 
 	cm.replicasDesired = NewReplicasDesiredStock(env, desiredConf, cm.replicaSource, cm.replicasLaunching, cm.replicasActive, cm.replicasTerminating)
 
+	//start with 1 active replica
+	rs := cm.replicaSource.(*replicaSource)
+	for i := 0; i < config.InitialNumberOfReplicas; i++ {
+		replicasActive.Add(NewReplicaEntity(rs.env, rs.kubernetesClient, rs.endpointsInformer, rs.Next(), rs.maxReplicaRPS))
```

Comment:
Created #68 to remove endpoints informer.  We can do it in a separate PR.  No action required, just FYI.

Reply:
Ok